### ISSUE TITLE
Parsing Rules

### ISSUE THREAD

**Issue by ericman314** [ericman314/UnitMath#4]
Currently in Math.js, the parser in Unit.js parses input such that the last two units below are parsed as `8.314 (J / mol) * K`, and are not equal to the first two:

```
8.314 J / mol / K
8.314 J / (mol K)
8.314 J / mol K
8.314 J / mol * K
```

I would like to propose a much simpler set of parsing rules that would make all these units equal:
- All parentheses and `*`'s are ignored.
- All units appearing before the first `/` are in the numerator.
- All units after the first `/` are in the denominator.
- Subsequent `/`'s are ignored.

Being able to parse multiple `*`, `/`, and parentheses is cool, but it *really seems beyond the scope of this library.*

If I am not mistaken, in Math.js, `math.eval` constructs compound units by parsing each individual node, then uses its own comprehensive parsing rules to build a tree. Only if a user calls `math.Unit()` directly would the `Unit.js` parser be invoked. So this change probably would not affect typical Math.js usage very much, I think.

**Comment by harrysarson** [comment]
> I would like to propose a much simpler set of parsing rules

👍 👍  to favouring simplicity

>  All parentheses are ignored, Subsequent `/`'s are ignored. (paraphrased)

This could lead to surprising behaviour. For example:

* `1 (m / s) * s` becomes `1 m / (s * s)` rather than `1 m`.
* `1 kg m / (m / s)` becomes `1 kg / s` rather than `1 kg s`.

These are silly examples so I would not argue that unitmath should _support_ them. I would argue that unitmath should not give the wrong answer in these cases. 

Instead, I think it would be most helpful to throw an error when attempting to parse strings containing `()` or more than one `/`. The error message might advise using a more powerful parsing tool (e.g. mathjs).

To summarise: I think a super simple parser is a fantastic idea. This simplicity should, however, be achieved by bailing when complicated inputs are encountered rather than trying to guess the user's intention.

**Comment by ericman314** [comment]
Good idea. We'll do it that way.

**Comment by ericman314** [comment]
Implemented via https://github.com/ericman314/UnitMath/commit/4b3c57059a2b9e5dff18214b7f65d54fa81b3ec2
